Homework2.py

```diff
@@ -0,0 +1,70 @@
+
+# Задание 1: программа считает сумму заказа
+
+m = int(input("Введите колличество рублей: "))
+n = int(input('введите колличество копеек: '))
+k = int(input('Количество покупок '))
+n = n / 100
+l = m + n
+all = l * k
+all_c = round(all // 1)
+all_p = round((all % 1) * 100)
+print('Спасибо, что сделали покупку у нас.')
+print('Общая цена ', all_c, ' рублей ', all_p, ' копеек.')
+
+# Задача 1
+# Выведите все элементы, которые меньше 5
+a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+for element in a:
+    if 5 < element < 25:
```

Comment:
this code contradicts with task description: if we should print numbers less than 5 this line isn't correct. If this line is correct we should fix description